### PR TITLE
separate my auctions list and searched auctions list

### PR DESCRIPTION
**Pull request by SzymonPuszcz**

### CODE REVIEW

File: Web/Models/HomeViewModels/HomeGetViewModel.cs [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+using System.Collections.Generic;
+using Allegrando.Web.Models.AuctionViewModels;
 
 namespace Allegrando.Web.Models.HomeViewModels
 {
     public class HomeGetViewModel
     {
         public AuctionCategoryNavViewModel AuctionCategories { get; set; }
+
+        public IEnumerable<AuctionListElementViewModel> Auctions { get; set; }
```
Comment (tomejzen) [inline Web/Models/HomeViewModels/HomeGetViewModel.cs]: So I would say better name for this property is LatestAuctions.

We may decide to create another section using collection of Auctions. (I mean section like 'Hottest acutions' or 'Ending in 5 minutes')

Comment (SzymonPuszcz) [reply Web/Models/HomeViewModels/HomeGetViewModel.cs]: Done

File: Web/Controllers/HomeController.cs [hunk 2]
```diff
@@ -6,29 +6,36 @@
...
             _categoryService = categoryService;
+            _auctionService = auctionService;
+            _applicationConfiguration = applicationConfiguration;
         }
 
         public async Task<IActionResult> Index()
         {
             var categories = await _categoryService.GetAllAsync();
             var categoriesModel = categories.Select(x => x.MapToGetViewModel()).ToList();
+            var auctions = await _auctionService.GetLatestAuctions(3);
```
Comment (krzysiekprzekwas) [inline Web/Controllers/HomeController.cs]: Magic number detected!!!!!!!!!!!!!!

File: Web/Views/Auctions/MyAuctionsTable.cshtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿@model IEnumerable<Allegrando.Web.Models.AuctionViewModels.AuctionListElementViewModel>
-@if (Model.Count() == 0)
+@if (!Model.Any())
```
[the remark this answers is not in the document]
Comment (krzysiekprzekwas) [reply Web/Views/Auctions/MyAuctionsTable.cshtml]: Yes yes

**Pull request merged**